internal/multierror/multierror.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
My understanding is that copyright applies to the creative work as a whole, which is the zap project; since that project started in 2016, all copyright starts there. (Imagine that Walt Disney updates a few of the line arcs in the original Mickey Mouse drawing. Those lines don't have a separate copyright date.)

Of course, I could easily be wrong.